### ISSUE TITLE
Staging Review finding: Screen offsets when navigating the page

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#128399]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits Management Tool Team 1
**Product:** Claim Status Tool
**Feature:** Document Submission Status

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

When navigating the Files section, using the Jaws screen reader and arrow navigation on a small screen (320px), when the screen reader cursor lands on the paragraph "You should provide a medical report..." in the Request for evidence alert, the screen offsets horizontally hiding ~50% of the content. 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="320" height="796" alt="screen offset" src="https://github.com/user-attachments/assets/44f42bec-b00f-43a0-90f3-03196cdf2bc3" />

---

https://github.com/user-attachments/assets/6a554cc8-a401-4606-bcbe-2e52dd98e1a9

</details>

## Recommended action

Unkown cause. Verify that layout for the tab panels, and that the rendered HTML structure is correct. 

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 1.4.10 Reflow - Level AA](https://www.w3.org/TR/WCAG22/#reflow)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
